micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -125,30 +117,110 @@
+  const localizationModule = `DIGIT_MDMS_${rawSchemaCode}`.toUpperCase();
+
+  const createLocalizationCode = (fieldName, fieldValue) => {
+    const upperFieldName = fieldName.toUpperCase();
+    const transformedValue = (fieldValue || "").replace(/\s+/g, "").toUpperCase();
+    return `${rawSchemaCode}_${upperFieldName}_${transformedValue}`.toUpperCase();
+  };
+
+  let localizationCodes = [];
+  if (data && data.data && localisableFields.length > 0) {
+    localizationCodes = localisableFields.map(field => createLocalizationCode(field.fieldPath, data.data[field.fieldPath]));
+    console.log("localizable fields constructed"+localizationCodes);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)